app/upgrades/v20/upgrades.go

```diff
@@ -328,13 +468,16 @@ func MigrateConsumerAdditionProposal(
 			),
 		)
 	} else {
-		// ConsumerAdditionProposal that was submitted, but not yet passed.
+		// MsgConsumerAddition that was submitted, but not yet passed.
 		// If the proposal is invalid, remove it.
 		// Otherwise, create a new consumer chain (MsgCreateConsumer), and
 		// replace the proposal's content with a MsgUpdateConsumer
 
-		metadata := metadataFromCAP(msg)
-		initParams, err := initParamsFromCAP(msg)
+		metadata := CreateConsumerMetadata(fmt.Sprintf("Chain with chain id %s", msg.ChainId), "TBA")
```

Comment:
@mpoke This would mean we would lose the `Title` and `Description` from `ConsumerAdditionProposal`s.